### ISSUE TITLE
New "source" release with pyproject.toml?

### ISSUE THREAD

**Issue by grantjenks** [louisabraham/pydivsufsort#38]
Thank you so much for these bindings! I am trying to use the library at work but we are forced to build things from source when importing them into our internal system. The current source release on PyPI uses Cython and Numpy in setup.py but doesn't declare these in `setup_requires`. I noticed that the latest code has added pyproject.toml with the requisite `build-system` `requires` section. I think this'll fix our builds of the library from source.

Would you cut a new source release to PyPI? Thanks again!

**Comment by louisabraham** [comment]
I don't know if the latest version is ready yet (we are currently working on main which is not ideal).

ccing @seanlaw 

Is it possible for you system to install from git? `pip install git+https://github.com/louisabraham/pydivsufsort`

You could also clone and try to build manually. What is your OS?

**Comment by seanlaw** [comment]
@louisabraham What do you think is necessary or outstanding for the latest version to be "ready"? From what I can tell, the wheels are being built correctly and needs to be uploaded to PyPI.

However, in @grantjenks case, it sounds like he cannot use the wheel and needs to install from source (maybe `sdist`?). So, that needs to get uploaded as well. 

@grantjenks Can you clarify precisely what you mean when you say "builds of the library from source"?

**Comment by louisabraham** [comment]
> What do you think is necessary or outstanding for the latest version to be "ready"?

I was waiting for your confirmation but it looks I am the late one :)

I think `pip install git+https://github.com/louisabraham/pydivsufsort` builds from source, just tested on macOS.

**Comment by seanlaw** [comment]
> I was waiting for your confirmation but it seems I am the late one :)

Maybe I'm missing something but I think Issue #37 is the final steps.

**Comment by grantjenks** [comment]
> Is it possible for your system to install from git?

Unfortunately, no. We have other workarounds but I figured since it was fixed at HEAD, I would just check on a new release.

> it sounds like he cannot use the wheel and needs to install from source (maybe sdist?)

Yes, that's correct. We import the sdist, build it ourselves, and then publish it to an internal PyPI-like server.

> I think pip install git+https://github.com/louisabraham/pydivsufsort builds from source, just tested on macOS.

Yes, that'll work. We just prefer now to pull the sdist from a version on PyPI (rather than from GitHub HEAD).

Thanks again for the bindings! The performance is really impressive.
